docs/pages/access-controls/device-trust/jamf-integration.mdx

```diff
@@ -0,0 +1,303 @@
+---
+title: Jamf Integration (Preview)
+description: Sync your Jamf inventory into Teleport
+---
+
+Device Trust Jamf integration lets you automatically sync your Jamf computer
+inventory into Teleport.
+
+Jamf service is a distinct `teleport` process that periodically reads your
+computer inventory from Jamf and syncs it to Teleport. It performs both
+incremental (called "partial") and full syncs, as well as removals from Teleport
+if a computer is removed from Jamf.
+
+Syncing devices from Jamf is an **inventory management** step, equivalent to
+automatically running the corresponding `tctl devices add` commands. If you want
+to automatically enroll synced devices on user login, refer to the
+[auto-enrollment](#optional-enable-auto-enrollment) section.
+
+See the [Device Trust guide](./guide.mdx) for fundamental Device Trust concepts
+and behavior.
+
+## Prerequisites
+
+(!docs/pages/includes/commercial-prereqs-tabs.mdx!)
+
+(!docs/pages/includes/device-trust/prereqs.mdx!)
+
+## Step 1/4. Create a Jamf user
+
+Create a readonly Jamf user for inventory sync.
+
+1. Access https://yourtenant.jamfcloud.com/accounts.html, replacing "yourtenant"
+   with your Jamf Pro account.
+
+2. Create a new Standard Account with the following settings:
+
+   - Username: teleport (change as desired)
+   - Access Level: Full Access
+   - Privilege Set: Custom
+   - Access Status: Enabled
+   - Password: (a strong password of your choice)
+   - Privileges:
+     - Advanced Computer Searches: Read
+     - Computers: Read
+
+   Take note of the user and password created here for the next step.
+
+   User account setup: ![Jamf user setup](
+   ../../../img/access-controls/device-trust/jamf-setup-1.png)
+
+   Privileges setup: ![Jamf privileges setup](
+   ../../../img/access-controls/device-trust/jamf-setup-2.png)
+
+## Step 2/4. Configure Jamf service
+
+Jamf inventory sync is performed by a separate `teleport` process, configured
+using the `jamf_service` key. It is recommended to run the service isolated from
+other Teleport processes, as it requires the Jamf credentials created in the
+step above.
+
+Save the following file as `/var/lib/teleport.yaml` and edit as needed:
+
+```yaml
+version: v3
+teleport:
+  # Necessary to write devices back to Teleport.
+  proxy_server: teleport.example.com:443 # CHANGEME
+
+jamf_service:
+  enabled: true
+  name: jamf
+  api_endpoint: https://yourtenant.jamfcloud.com # CHANGEME
+  username: teleport                             # CHANGEME
+  password_file: /var/lib/teleport/jamf_password.txt
+
+auth_service:
+  enabled: false
+
+proxy_service:
+  enabled: false
+
+ssh_service:
+  enabled: false
+```
+
+Change the following settings, as appropriate:
+
+- teleport.proxy_server
+- jamf_service.api_endpoint
+- jamf_service.username
+
+Finally, write your Jamf password to the `/var/lib/teleport/jamf_password` file:
+
+```code
+$ nano /var/lib/teleport/jamf_password # or use your favorite editor
```

Comment:
I didn't want to echo the password because if the user is not careful it'll be recorded in their shell history. It's also difficult to explain that in a succinct manner, so I picked a friendly-ish editor instead.

Let me know if you want this changed anyway.

Reply:
Fair point, leave it as is 👍